src/coreclr/tools/ILVerification/ILImporter.Verify.cs

```diff
@@ -2582,6 +2582,8 @@ void ImportLocalAlloc()
 
             var size = Pop();
 
+            FatalCheck(_stackTop == 0, VerifierError.LocallocStackNotEmpty);
```

Comment:
```suggestion
            Check(_stackTop == 0, VerifierError.LocallocStackNotEmpty);
```
We should be able to continue verifying the current method after encountering this.